docs/flow/modin/core/execution/unidist/generic.rst

```diff
@@ -0,0 +1,15 @@
+:orphan:
+
+Generic Unidist-based members
+=============================
+
+Objects which are storage format agnostic but require specific Unidist implementation
+are placed in ``modin.core.execution.unidist.generic``.
+
+Their purpose is to implement certain parallel I/O operations and to serve
+as a foundation for building storage format specific objects:
+
+* :py:class:`~modin.core.execution.unidist.generic.partitioning.GenericUnidistDataframePartitionManager` -- implements parallel :meth:`~modin.core.execution.unidist.generic.partitioning.GenericUnidistDataframePartitionManager.to_numpy`.
```

Comment:
```suggestion
```
Let's just remove this and add UnidistIO class bellow as for GenericUnidistDataframePartitionManager.